apps/meteor/ee/app/license/server/airGappedRestrictionsCheck.ts

```diff
@@ -0,0 +1,16 @@
+import { License, AirGappedRestriction } from '@rocket.chat/license';
+import { Statistics } from '@rocket.chat/models';
+
+export async function checkAirGappedRestrictions(): Promise<void> {
+	if (License.hasModule('unlimited-presence')) {
+		AirGappedRestriction.removeRestrictions();
+		return;
+	}
+
+	const { statsToken: encryptedStatsToken } = (await Statistics.findLast()) || {};
```

Comment:
Do we need _everything_ from the statistics record? Bet we should create a new model method that only returns the last statsToken. wdyt?

Reply:
This file doesn't exist anymore. I moved the check to the statistics cron and getting the token from there.